contrib/pg_tde/pg_tde--1.0-rc.sql

```diff
@@ -455,12 +455,6 @@ END
 $BODY$
 LANGUAGE plpgsql;
 
--- Table access method
-CREATE FUNCTION pg_tdeam_basic_handler(internal)
-RETURNS table_am_handler
-LANGUAGE C
-AS 'MODULE_PATHNAME';
```

Comment:
Don't we need to add dropping this to an upgrade script now that we have release the RC?